From 92e6d17fc94ddf61f9a4588783d50a0c423f84f2 Mon Sep 17 00:00:00 2001
From: d2j3 <donkeyjjlove@protonmail.com>
Date: Mon, 29 Sep 2025 20:55:57 +0200
Subject: [PATCH] =?UTF-8?q?1=20file(s)=20staged:=20[=CE=94]=20Zakres=20-?=
 =?UTF-8?q?=20files:=201=20(README.md)=20-=20typ:=20auto=20(pre-diff)?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

[Φ/Ψ] Mozaika (semantyka kodu)
- Align(mean .py): 0.00
- Hint: brak wpływu na strukturę sąsiednich kafli

[AST] Deltas (staged .py)
- S: 0  H: 0
- uwagi: wartości przybliżone (heurystyki)

[Dokumentacja]
- decyzja: NO-OP

Meta
- Generated-by: pre-diff/AST-mosaic @ 2025-09-29 18:55:58Z

…
---
 gui/mosaic/README.md | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/gui/mosaic/README.md b/gui/mosaic/README.md
index 48258e6..e84217c 100644
--- a/gui/mosaic/README.md
+++ b/gui/mosaic/README.md
@@ -21,8 +21,7 @@
 * [8. Pozytywne/negatywne meta-tagi (kontrola generacji)](#8-pozytywnenegatywne-meta-tagi-kontrola-generacji)
 * [9. Przepisy (recipes): wstrzymanie i wymuszenie wzorca](#9-przepisy-recipes-wstrzymanie-i-wymuszenie-wzorca)
 * [10. Uruchomienia i CLI](#10-uruchomienia-i-cli)
-* [11. Wstawki do GitHuba (obrazy i struktura)](#11-wstawki-do-githuba-obrazy-i-struktura)
-* [12. Najczęstsze pytania](#12-najczęstsze-pytania)
+* [11. Najczęstsze pytania](#12-najczęstsze-pytania)
 
 ---
 
-- 
2.45.1.windows.1

